deployments/curity/apigateway/openresty/Dockerfile

```diff
@@ -0,0 +1,32 @@
+
+FROM openresty/openresty:1.25.3.2-jammy
+
+#
+# Install luarocks, and set git options if required
+#
+RUN apt-get update && apt-get install git -y
+RUN git config --global url."https://".insteadOf git:// && \
+    git config --global advice.detachedHead false
+
+#
+# Install an example CORS plugin 
+#
+COPY openresty/cors.lua \
+     openresty/cors-*.rockspec \
+     /tmp/cors/
+RUN cd /tmp/cors && luarocks make cors-*.rockspec \
+     && cd / && rm -r /tmp/cors
+
+#
+# Install the OAuth Proxy plugin and its dependencies
+#
+COPY resources/plugin/*.lua \
```

Comment:
rockspec is used since it installs dependencies. To get the official token handler OAuth proxy components you must be signed into the developer portal, so the official luarocks site cannot be used currently.